components/Link.jsx

```diff
@@ -1,22 +1,32 @@
 import React, { PropTypes } from 'react'
+import { Link as ReactRouterLink } from 'react-router'
```

Comment:
I might be wrong, but wouldn't that import react-router in any project even if this dependency is not used?

Maybe it's better to create a RouterLink that will extend simple Link, so Checkout might still use router-less version and other projects more complex one with react-router dependency?